src/main/java/com/useful/ucars/ucars.java

```diff
@@ -683,21 +683,22 @@ public final Boolean isBlockEqualToConfigIds(List<String> rawIds, Block block) {
 		for (String raw : rawIds) {
 			final String[] parts = raw.split(":");
 			if (parts.length < 1) {
-			} else if (parts.length < 2) {
-				if (parts[0].equalsIgnoreCase(block.getType().name())) {
+			} else if (parts.length < 2) { //New configs and blocknames
```

Comment:
This doesn't quite seem like the right check since the old format may not have specified the :data part but it's probably fine - just means some people will need to edit the configs a little

Reply:
I suspect you mean something like WOOD in the config and it therefore working on all types of wood?

Reply:
Yeah that's what I mean